components/eamxx/docs/developer/standalone_testing.md

```diff
@@ -21,7 +21,40 @@ configurations. We use several types of tests
   the [ctest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) command.
 
 We also support a `test-all-scream` configuration that runs all of the
-standalone tests for an EAMxx configuration.
+standalone tests for an EAMxx configuration. Note, your current machine
+must be known to EAMxx before `test-all-scream` will work. A machine can
+be made known to EAMxx by editing the eamxx/scripts/machines_specs.py files.
+There are some instructions on what to do at the top of this file.
+
+`test-all-scream` has a good help dump
+```
+% cd $scream_repo/components/eamxx
+% ./scripts/test-all-scream -h
+```
+
+If you are unsure of the cmake configuration for you development cycle, one
```

Comment:
We could mention the `--config-only` flag, rather than ctrl-C.

Reply:
We should also mention that they may have to run `eval $(./scripts/scream-env-cmd $mach)` before running make in the build folder...

Reply:
> We could mention the --config-only flag, rather than ctrl-C.

I forgot about this option. I just tried it and unfortunately you still have to wait over a minute:
```
real	1m20.001s
```

I will make note of the need to load the scream env.

Reply:
Ah, cause it has to actually run cmake, while with ctrl-C you kill it after it shows the cmake line. That makes sense.

That said, we often need to build/run just one exec. For that, we need to eventually run cmake anyways, so we should advertise the `--config-only` option, I think.